.eslintrc.cjs

```diff
@@ -29,5 +29,5 @@ module.exports = {
     ],
   },
   // ignore js files
-  ignorePatterns: ['.eslintrc.cjs', 'vite.config.ts', '**/*.js'],
+  ignorePatterns: ['.eslintrc.cjs', 'vite.config.ts', '**/*.js', 'src/tests'],
```

Comment:
Why are we not linting the tests?

Reply:
Because there are some lints that are not sonidered for testing purposes, like:
```
/Users/adriansacristan/Documents/code/CPR/navigator-admin-frontend/src/tests/utilsTest/render.tsx
  25:7  warning  Fast refresh only works when a file only exports components. Move your component(s) to a separate file  react-refresh/only-export-components
```

Reply:
@diversemix have you got a better solution in mind?

Reply:
Pending to discuss about it in the future